Marlin/src/feature/e_parser.h

```diff
@@ -79,7 +79,6 @@ class EmergencyParser {
 
   static void update(State &state, const uint8_t c);
 
-private:
   static bool enabled;
```

Comment:
It would be better to leave this private, and add a static accessor to return it's state. That removes the possibility that somebody makes a change outside the EmergencyParser class that decides to actually disable it.

Something like this (although formatting isn't correct, don't apply this directly)

```
public:
   static bool is_enabled() { return enabled; }
private:
   static bool enabled;
```

Reply:
This is just general C++ guidelines. The more we encapsulate roles inside the class, the more we are able to reason about and test a class in isolation, without wondering whether the rest of the code is mucking with it somehow.